Apply loading indicator

lanieerts opened the pull request.

lanieerts committed:
Seperate reducer about comment and modify query.

lanieerts committed:
Modify detail of query for readMoreComments

lanieerts committed:
Modify url of action for createComment.

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

pjhjohn committed:
Rebase commit of upstream/master.
Introduce react-router-bootstrap to avoid page refresh

Removed legacy style for Tab. Removed navbar radius by overriding style

Resolved #32 : Implemented Sticky Footer

Reference : https://getbootstrap.com/examples/sticky-footer

Disabled default visibility of ReduxDevTools

CRLF -> LF

reference : http://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

Comment, PostDetail, Post re-styled with react-bootstrap

* Enabled well class in _bootstrap.scss

UI Enhancement using Panel Component

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' of https://github.com/lanieerts/jsonplaceholder-client

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Add loading indicator.

lanieerts committed:
Merge remote-tracking branch 'upstream/master'

lanieerts committed:
Merge branch 'master' into apply-loading-indicator

pjhjohn commented on src/containers/About/Issues.js:
```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
 
   render() {
-    if(!this.props.contributors.length) return (<div>LOADING</div>);
+    const loading = (<center> <CircularProgress /> </center> );
```
`<center>` tag has been deprecated from HTML
http://stackoverflow.com/questions/11478067/replacement-for-center

pjhjohn commented on src/containers/About/Issues.js:
```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
-    }
+    },
+    initialized: false,
+    loadingSignal: false
```
The meaning of `Signal` is not clear and also redundent.
`loading` or `isLoading` are more clear

lanieerts committed:
Complete issues of PR #58.

Replace 'loadingSignal' to 'isLoading'.

Replace center tag to div styles.

pjhjohn commented on src/containers/About/Issues.js:
```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
+    this.setState({isLoading: true});
     const newFilter = Object.assign(this.state.filter, { [key] : value });
     if(!value) delete newFilter[key];
     this.setState({ filter: newFilter });
```
setState는 한번에 하는게 어떨까요?

pjhjohn commented on src/containers/About/Issues.js:
```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
 
   render() {
-    if(!this.props.contributors.length) return (<div>LOADING</div>);
+    const loading = (<div style={{textAlign:`center`, width:`100%`}}> <CircularProgress /> </div>);
```
`CircularProgress` 자체가 `<div><svg/></div>` 처럼 구현되는데, 그냥
`<CircularProgress style={{marginLeft: 'auto', marginRight: 'auto'}} />` 같은건 안되나요?

lanieerts committed:
Modify 2 issues about PR #58.
1. Remove div tag for CircularProgress.
2. Combine setState function in updateFilter function.

lanieerts committed:
Modify loading to progress & isLoading to loading.
